src/Raven.Studio/wwwroot/App/views/database/tasks/ongoingTasks.html

```diff
@@ -336,10 +336,71 @@ <h5 class="tasks-list-item subscription no-text-transform"><i class="icon-subscr
+                                                                </span>
+                                                            </small>
+                                                            <span data-bind="visible: nextBatchStartingPointDocumentFound">
+                                                                <small>Matching document: </small>
+                                                                  <a data-bind="attr: { href: urlForNextBatchStartingPointDocument() }" target="_blank"
+                                                                     title="Click to open document"><small data-bind="text: documentIDForNextBatchStartingPoint"></small></a>
+                                                            </span>
+                                                        </span>
+                                                    </span>
+                                                    <span data-bind="visible: !changeVectorForNextBatchStartingPoint()">
+                                                       N/A
+                                                    </span>
+                                                </div>
+                                            </div>
+                                            <div class="property-item">
+                                                <div class="property-name">Last Change Vector Acknowledged: </div>
+                                                <div class="property-value text-details">
+                                                    <span data-bind="visible: lastChangeVectorAcknowledged()">
+                                                        <span data-bind="foreach: lastChangeVectorAcknowledgedFormatted">
+                                                                <span class="badge badge-default" data-bind="text: shortFormat"></span>
+                                                        </span>
+                                                        <span>
+                                                            <small data-bind="visible: !lastAcknowledgedDocumentFound()">
+                                                                <a href="#" data-bind="click: _.partial(tryGetDocumentIDFromChangeVector, 'LastAcknowledgedByClient'), visible: !spinners.documentIDForLastChangeVectorAcknowledgedLoading()" 
+                                                                   title="Search for the document that matches this change-vector">
+                                                                     <i class="icon-search"></i>
+                                                                </a>
+                                                                <span data-bind="visible: spinners.documentIDForLastChangeVectorAcknowledgedLoading">
+                                                                    &nbsp;&nbsp;&nbsp;<i class="btn-spinner"></i>
```

Comment:
nbsp nbsp

Reply:
removed one 'nbsp' but left one 'nbsp' as it was mandatory